ec2/spark_ec2.py

```diff
@@ -349,6 +350,15 @@ def launch_cluster(conn, opts, cluster_name):
     if opts.identity_file is None:
         print >> stderr, "ERROR: Must provide an identity file (-i) for ssh connections."
         sys.exit(1)
+
+    if not os.path.exists(opts.identify_file):
+        print >> stderr, "ERROR: The identity file(%s) doesn't exist." % opts.identity_file
```

Comment:
Formatting nitpick:

```
print >> stderr, "ERROR: The identity file '{f}' doesn't exist.".format(f=opts.identity_file)
```

Even though there are other places in `spark_ec2.py` where we use `%`, I think [`format()` is generally preferable](http://stackoverflow.com/a/12382738/877069).